fix(scenario-test.yaml): install `xmlschema` in `scenario-test.yaml`

vish0012 opened the pull request:
## Description

This PR fixed https://github.com/autowarefoundation/autoware/issues/6118.

https://github.com/autowarefoundation/autoware/actions/runs/14921050636/job/41916359477?pr=6122#step:17:80

## How was this PR tested?

https://github.com/vish0012/autowarefoundation/actions/runs/14929242188/job/41941062796


## Notes for reviewers

None.

## Effects on system behavior

None.

youtalk reviewed (changes requested):
I've changed the base branch to #6122. Please refine the title and minimize the changes.

youtalk reviewed (commented):
Please remove meaningless linefeed changes and fix the `precommit` errors.

Also the `test_results.log` file still isn’t found. Should we just ignore that?
https://github.com/autowarefoundation/autoware/actions/runs/14960756013/job/42022310695?pr=6129#step:18:40
```
ERROR: test_results.log is missing! The simulation might have crashed.
```

vish0012 reviewed (commented):
removed meaning less line space in script

youtalk reviewed (commented):
https://github.com/autowarefoundation/autoware/pull/6129/commits/beed4282afe15d939141b497126fe1101d287554 is wrong.

The original branch https://github.com/autowarefoundation/autoware/pull/6122/files has no redundant linefeed changes. Please remain them.

vish0012 commented:
> Please remove meaningless linefeed changes and fix the `precommit` errors.
> 
> Also the `test_results.log` file still isn’t found. Should we just ignore that? https://github.com/autowarefoundation/autoware/actions/runs/14960756013/job/42022310695?pr=6129#step:18:40
> 
> ```
> ERROR: test_results.log is missing! The simulation might have crashed.
> ```

Thank you sensei, Thank you for the feedback.

 I’ve removed the unnecessary linefeed changes and fixed the pre-commit errors (including trailing-whitespace).
Pre commit is fixed

 Regarding the missing test_results.log, it appears that the simulation might have exited early or failed silently during execution. may we can ignore for now, but i will check later simulation side more if i find some cause.

youtalk commented:
> [beed428](https://github.com/autowarefoundation/autoware/commit/beed4282afe15d939141b497126fe1101d287554) is wrong.
> 
> The original branch https://github.com/autowarefoundation/autoware/pull/6122/files has no redundant linefeed changes. Please remain them.

@vish0012 Please do them.

vish0012 reviewed (commented):
revert back changes

youtalk commented:
See https://github.com/autowarefoundation/autoware/pull/6129/files. There are lots of meaningless linefeeds and indents. Please remove them all.
I think you are always using GitHub online editor. Since it is very simple and difficult to edit in detail. Please use your local editor instead and git commands. cc @mitsudome-r

vish0012 commented:
> See https://github.com/autowarefoundation/autoware/pull/6129/files. There are lots of meaningless linefeeds and indents. Please remove them all. I think you are always using GitHub online editor. Since it is very simple and difficult to edit in detail. Please use your local editor instead and git commands. cc @mitsudome-r

> Thank you for your guidance sensei. I've made the changes as suggested, removing unnecessary linefeeds and indents using a local editor and git commands. If there are still any issues or errors, please let me know—I’ll promptly correct them. Thank you again for your patience and support!

youtalk commented on .github/workflows/scenario-test.yaml:
```diff
@@ -41,14 +37,16 @@ jobs:
           rosdep install --from-paths src --ignore-src -r -y
         shell: bash
-
+      - name: Downgrade xmlschema
+        run: |
+          pip3 install --user xmlschema==3.4.5
+        shell: bash
```
Please eliminate the modifications except here. In other words, please stop removing line feeds to be minimum modification PR.

vish0012 replied:
Thank you sensei , for your patient i have eliminate unnecessary comment, please let me know if still there an issue :), i will try again

vish0012 committed:
feat: fixed the xml schema

Signed-off-by: vish0012 <vishalchhn42@gmail.com>

vish0012 committed:
fix: reduces unnessary changes

Signed-off-by: vish0012 <vishalchhn42@gmail.com>

mitsudome-r committed:
Merge branch 'fix/output-directory' into main

vish0012 committed:
fix: line spaces

Signed-off-by: vish0012 <vishalchhn42@gmail.com>

youtalk reviewed (approved):
Finally LGTM. Well done!
